spharpy/plot/spatial.py

```diff
@@ -813,3 +804,45 @@ def __call__(self, value, clip=None):
         # simple example...
         x, y = [self.vmin, self.midpoint, self.vmax], [0, 0.5, 1]
         return np.ma.masked_array(np.interp(value, x, y))
+
+
+def coordinates2latlon(coords: pf.Coordinates):
```

Comment:
Do we need that function, or could it be replaced by coordinates from `pyfar.Coordinates`? The angles look like the **spherical_elevation** coordinates system from pyfar?